src/Catty/ViewController/Continue&New/MaintainObject/MaintainSounds/SoundsTableViewController+SelectFile.swift

```diff
@@ -0,0 +1,79 @@
+/**
+ *  Copyright (C) 2010-2021 The Catrobat Team
+ *  (http://developer.catrobat.org/credits)
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  An additional term exception under section 7 of the GNU Affero
+ *  General Public License, version 3, is available at
+ *  (http://developer.catrobat.org/license_additional_term)
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see http://www.gnu.org/licenses/.
+ */
+import MobileCoreServices
+
+extension SoundsTableViewController {
+
+    @objc
+    func showSoundsSelectFile() {
+        let documentPicker = UIDocumentPickerViewController(
```

Comment:
`init(documentTypes:in:)` is marked as deprecated - please use the alternatives like e.g. [init(forOpeningContentTypes:)](https://developer.apple.com/documentation/uikit/uidocumentpickerviewcontroller/3566732-init). Also for example the import of .wav files on the current iOS version does not work.